.rubocop.yml

```diff
@@ -157,6 +157,10 @@ Style/TrailingCommaInArrayLiteral:
 Rails/FilePath:
   EnforcedStyle: slashes
 
+Rails/UniqueValidationWithoutIndex:
```

Comment:
This popped up from editing the 1reaction.rb` file, I didn't want to tackle it in this PR so I excluded it from Rubocop. It is a new cop that came with our recent gem upgrade

Reply:
Nice, will you do a follow-up PR yourself or should we take note of this somewhere?

Reply:
Ah! I was working on a PR to fix some new rubocop violations and put those new ones in the todo. I'll submit it shortly

Reply:
https://github.com/thepracticaldev/dev.to/pull/6904